doc/minetestserver.6

```diff
@@ -1,8 +1,7 @@
-.\" Minetestserver man page
 .TH minetestserver 6 "10 September 2013" "" ""
 
 .SH NAME
-minetestserver \- Minetest server
+minetestserver \- Minetest server-only binary
```

Comment:
It makes it clear that the main executable also includes a server that you can use instead.  What do you mean by "misses the point of the description"?  This should really be a alias for the main man page to remove the duplication though...